softvis3d-frontend/src/components/sidebar/FolderContent.tsx

```diff
@@ -77,20 +78,23 @@ export default class FolderContent extends React.Component<NodeListProps, NodeLi
             return [];
         }
 
-        const {sceneStore} = this.props;
-
         let folderElements: JSX.Element[] = [];
-        for (let child of folder.children) {
-            folderElements.push(
-                <FolderContentElement
-                    key={child.id}
-                    element={child}
-                    isSelected={child.id === sceneStore.selectedObjectId}
-                    sceneStore={sceneStore}
-                />
-            );
+        for (let child of folder.children.filter((e) => !e.isFile)) {
```

Comment:
Moved sorting code to TreeElement class which reduces the logic in FolderContent. Nice sort function. As a java developer, this is no really easier to understand ;-) but i get used to it slowly.